benchmark/strong_scaling_incompressible_model.jl

```diff
@@ -4,26 +4,48 @@ using Benchmarks
 
 # Benchmark parameters
 
-Nx = 256
-Ny = 256
-Nz = Nx
+decomposition = Slab()
 
 ranks = (1, 2, 4, 8, 16)
 
-# Run and collect benchmarks
+grid_size(R, decomposition) = (256, 256, 256)
+
+rank_size(R, ::Slab) = (1, R, 1)
+rank_size(R, ::Pencil) = Int.((1, √R, √R))
+
+# Run benchmarks
 
 print_system_info()
 
-for r in ranks
-    @info "Benchmarking distributed incompressible model strong scaling [N=($Nx, $Ny, $Nz), ranks=$r]..."
-    julia = Base.julia_cmd()
-    run(`mpiexec -np $r $julia --project strong_scaling_incompressible_model_single.jl $Nx $Ny $Nz`)
-end
+# for R in ranks
```

Comment:
Do we want to keep these comments or remove them?

Reply:
Ah woops, meant to uncomment this. Thanks for catching it!